### PR TITLE
Added one more known issue

### PR DESCRIPTION
**Pull request by dplotnikov**
as described in https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2620

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Known-issues.md [hunk 1]
```diff
@@ -245,6 +245,10 @@ This article lists the known issues for Microsoft Teams, by feature area.
 |:-----|:-----|:-----|:-----|
 |Phone number blocked|Phone number blocked: Deleting the resource account before removing the phone number will block the phone number.|Contact Microsoft support to release the telephone number.|5/18/2019|
 
+|**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
+|:-----|:-----|:-----|:-----|
+|Unable to configure unanswered calls to go to voicemail in the desktop app|When a user is in Teams-only mode, their Exchange mailbox is on premises and they access Settings > Calls in the Teams desktop app, they cannot select the options to forward calls to voicemail (unanswered calls or all calls). If either option was already configured, the act of opening the Calls tab of Settings will disable forwarding to voicemail.|Configure forwarding to voicemail using the Teams mobile app.|5/27/2019|
```
Comment (JohanFreelancer9) [inline Teams/Known-issues.md]:
```suggestion
|Unable to configure unanswered calls to go to voicemail in the desktop app|When a user is in Teams-only mode, their Exchange mailbox is on premises and if they access **Settings** > **Calls** in the Teams desktop app, they cannot select the option to forward calls to voicemail (unanswered calls or all calls). If either option was already configured, opening the Calls tab in Settings will disable forwarding to voicemail.|Configure forwarding to voicemail using the Teams mobile app.|5/27/2019|
```

**Comment by dplotnikov** [comment]
@scanum it's ready for the merge

**Comment by LolaJacobsen** [comment]
@marclauinger - please review.

**Comment by scanum** [comment]
@marclauinger Any updates on this PR? Thanks

**Comment by scanum** [comment]
@LolaJacobsen Could you please merge? Thanks!

**Pull request merged**